notebooks/generate.ipynb

```diff
@@ -25,12 +25,9 @@
     "# Uncomment to clone and install autodoc from GitHub\n",
     "# !pip uninstall -y autora-doc\n",
     "# !git clone https://github.com/AutoResearch/autodoc.git\n",
-    "# !pip install -e \"./autodoc[cuda,train]\"\n",
+    "# !pip install \"./autodoc[cuda,train]\"\n",
     "\n",
-    "# Login to Huggingface since access to the model repo is private\n",
-    "# 1) Request access through: https://ai.meta.com/resources/models-and-libraries/llama-downloads/\n",
-    "# 2) Get a Huggingface token from: https://huggingface.co/settings/token (use same email as above)\n",
-    "# !huggingface-cli login --token <your HF token>"
```

Comment:
Do you want the user to hard code the HF token into the notebook or pull it from an env variable?

Reply:
This was removed, no login required since we switched to a public model